persistence/data_models_json_fields.go

```diff
@@ -137,23 +137,26 @@ type WorkerTaskBackoffInfoJson struct {
 	FirstAttemptTimestampSeconds int64 `json:"firstAttemptTimestampSeconds"`
 }
 
-type WorkerTaskInfoJson struct {
+type ImmediateTaskInfoJson struct {
+	// used when the `task_type` is waitUntil or execute
 	WorkerTaskBackoffInfo *WorkerTaskBackoffInfoJson `json:"workerTaskBackoffInfo"`
+	// used when the `task_type` is localQueueMessage
+	LocalQueueMessagePayload *xdbapi.EncodedObject `json:"localQueueMessagePayload"`
```

Comment:
I think this refactor is big enough for a PR. It's not too late to do it later when you are implementing the feature -- I feel like we may not need the payload. Probably dedupIds/queueNames are enough if necessary.

Reply:
True. I modifed this field to be :
```
type LocalQueueMessageInfoJson struct {
	QueueName string    `json:"queueName"`
	DedupId   uuid.UUID `json:"dedupId"`
}
```